### ISSUE TITLE
Missing alt text on viewer images

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#7367]
**What**
The viewer images are missing alt text

**Where**
[wellcomecollection.org/works/f242ps6b/items?canvas=29](https://wellcomecollection.org/works/f242ps6b/items?canvas=29)

<img width="218" alt="Screenshot 2021-11-24 at 14 35 47" src="https://user-images.githubusercontent.com/6051896/143258191-1fc34e7c-ed6f-44a0-9f08-b436d717e7a9.png">

The alt attribute is usually populated by text provided by the OCR service for the IIIFCanvas.
 
It looks like the label we were relying on to extract the OCR service from the IIIFCanvas has changed, so we are no longer retrieving the text.
